src/components/Items/ProfileContact.js

```diff
@@ -67,7 +67,7 @@ const ProfileContact = ({
         )}
         {select && (
           <View testID="selected">
-            <MemoIc_Checklist />
+            <IconChecklist />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `IconChecklist` in the component is correct. Ensure that the renamed component `IconChecklist` maintains the same functionality as the old `MemoIc_Checklist`.
```diff
- <MemoIc_Checklist />
+ <IconChecklist />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->